### PR TITLE
cloud-mlcloud7: add mitaka template.

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by kbaikov** [comment]
+1

File: scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml [hunk 1]
```diff
@@ -16,3 +16,4 @@
         - 'cloud-mkcloud{version}-job-tempestfull'
         - 'cloud-mkcloud{version}-job-upgrade'
         - 'cloud-mkcloud{version}-job-xen'
+        - 'cloud-mkcloud{version}-job-mitaka'
```
Comment (vuntz) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml]: Just wondering: isn't the list sorted alphabetically? (before the patch ;-))

Comment (aplanas) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml]: very true, fixing.

**Comment by aplanas** [comment]
* Ordered the template list (mitaka is now hidden inside the list)

**Comment by dirkmueller** [comment]
+1

**Pull request merged**